### DISCUSSION TITLE
how do i use discord webhooks

### DISCUSSION THREAD

**Discussion by pigwin-3** [ORelio/Minecraft-Console-Client#1885] [Q&A] [answered]
i am not quite able to use this discord webhook https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/config/ChatBots/DiscordWebhook.cs can anyone help me?

**Comment by Daenges** [comment, marked as answer]
First download the file and save it as `DiscordWebhook.cs` in the same folder as your client.

Second you need to create a webhook on your discord server and copy its link.

Third you load the script in the client with `/script discordwebhook`

Fourth you enter your webhook url with `/dw changeurl YOURURL`

Then the webhook should be working.

**Reply by pigwin-3** [reply]
thanks

**Reply by ghost** [reply]
Hi, when I follow these steps, all is well until:

[MCC] [DiscordWebhook] Saved message to cache.
[MCC] [DiscordWebhook] Send webhook request to Discord.
[MCC] [DiscordWebhook] An error occured while posting messages to Discord! (Enable Debug to view it.)
[MCC] [DiscordWebhook] Requested Link https://crafatar.com/avatars/3a71e91a99ff4987bfa5b8d225f8ece9?size=100&scale=4&default=MHF_Steve&overlay; Username Alchemica; message: tt ; error: System.Net.WebException: The request was aborted: Could not create SSL/TLS secure channel.
   at System.Net.WebClient.UploadValues(Uri address, String method, NameValueCollection data)
   at System.Net.WebClient.UploadValues(String address, NameValueCollection data)
   at ScriptLoader.Script.HTTP.Post(String url, NameValueCollection pairs)
   at ScriptLoader.Script.DiscordWebhook.SendWebhook(Message msg)
[MCC] [DiscordWebhook] Cleared cache!

**Reply by Daenges** [reply]
Oh which platform are you running the client?
There is a SSL/TLS fix for [mono](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#how-to-use).

**Reply by ghost** [reply]
I am using Windows 10

**Reply by Daenges** [reply]
> I am using Windows 10

Found a fix for your issue. You can copy the code from #2114.
